assets/css/index.css

```diff
@@ -210,4 +210,22 @@ hr {
   height:50%;
   background-color:white;
   margin-left: 20vw;
-  }
\ No newline at end of file
+  }
+
+  .about__main {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    margin: 0 100px;
+  }
+  
+  .about__main__container {
+    width: 200px;
+    text-align: center;
+  }
+
+@media only screen and (max-width: 1100px) {
```

Comment:
@elixier01 Hi, could you explain what's going on in this line to me please?
Is it, for only screen (therefore exclude old browsers, printing devices etc) where the max-width of the screen is 1100px, set the flex basis to 50%. So if the width is 200px, we are saying, set the width of the divs with class about__main__container to 100px. Is that correct?

Reply:
@BarryGallagher Yep you are pretty much there. So it's a media query that will come into affect with screen sizes smaller than 1100px. Flex-basis sets the size of a flex item. So we are setting it to 50% of it's original size. The reason we did this was because as you moved from a desktop screen to smaller screens, the items in the 'About' section would go from 4 across, to 3 with 1 on the next row, then 2x2. We wanted it to go from 4 across straight to 2x2 so this was the outcome of it! Sorry for the long winded response but hopefully that explains it all!

Reply:
Great, long winded is good. So each about__main__container  is 200px (800px in total) `margin: 0 100px` means margin left 100px and margin right 100px (200px), total = 1000px, so not sure why we chose 1100px.
Anyway, I understand what going on, once we reach a certain width, make the items 50% of the parent container, therefore, you'll end up with 2*2